.github/workflows/integration.yml

```diff
@@ -0,0 +1,56 @@
+name: Playwright Integration Tests
```

Comment:
How do we make sure this is a required check? Let me know if you don't have the correct GitHub permissions to do that

Reply:
Not sure I know how either. Let's get this in once you're happy with it and I can figure out how to make it required then